.github/actions/docker-run/action.yml

```diff
@@ -64,5 +61,4 @@ runs:
         run: |
           cp -r /github_workspace/* /usr/app/
           cd /usr/app/
-          rm -rf tt_metal tt_eager
```

Comment:
details: Just a note - it's actually better for our wheel validation that we don't have these folders there, but no need

Reply:
i can't actually get it to build with that line there so i wasn't sure whats the issue there

Reply:
I think that's probably because it needs the files to build? So actually yea - should delete for now